src/Stryker.Core/Stryker.Core/TestRunners/CoverageRunResult.cs

```diff
@@ -43,9 +41,6 @@ public class CoverageRunResult
 
         public CoverageConfidence Confidence { get; private set; }
 
-        public IReadOnlyCollection<int> LeakedMutations  => _mutationFlags
```

Comment:
Note: opportunistic improvement. This property is not used and should have been removed earlier.